.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: Build & Test
+name: CI
```

Comment:
This isn't a _great_ name. But the only place it shows up is here:
![Screen Shot 2022-04-15 at 11 47 27 AM](https://user-images.githubusercontent.com/1930213/163591773-c7ae3cfd-116a-420a-a04e-98f32c09f320.png)

Where I wanted something short+sweet to focus on the job that actually failed